### PR TITLE
upgrade: increase coverage for upgrade.factory.js

### PR DESCRIPTION
**Pull request by Itxaka**

### CODE REVIEW

File: assets/app/data/crowbar/services/upgrade.factory.spec.js [hunk 1]
```diff
@@ -9,13 +9,43 @@ describe('Upgrade Factory', function () {
...
+                'success': true
+            },
+            'schema_migration': {
+                'success': true
+            },
+            'crowbar_init': {
+                'success': true
+            }
+        },
+        mockedConnectDatabaseServerResponse = {
```
Comment (santiph) [inline assets/app/data/crowbar/services/upgrade.factory.spec.js]: How is this different to `mockedNewDatabaseServerResponse` from line 12?

Comment (Itxaka) [reply assets/app/data/crowbar/services/upgrade.factory.spec.js]: Its not, but for the cost of one var we can visually separate them in the tests for easier comprehension of what it refers to.

Comment (skazi0) [reply assets/app/data/crowbar/services/upgrade.factory.spec.js]: I would change it to `mockedConnectDatabaseServerResponse = mockedNewDatabaseServerResponse`. This gives us the effect of separation as you mentioned but also shows that at the moment the responses are identical.

**Review by skazi0 — CHANGES_REQUESTED** [review]
small nit as commented

File: assets/app/data/crowbar/services/upgrade.factory.spec.js (hunk 1 above)
Comment (Itxaka) [reply assets/app/data/crowbar/services/upgrade.factory.spec.js]: done!

**Comment by Itxaka** [comment]
Need rebase after https://github.com/crowbar/crowbar-ui/pull/104 is in

**Review by skazi0 — APPROVED** [review]

**Comment by santiph** [comment]
No issues found, approved

**Pull request merged**